tests/src/Kernel/ResolverBuilderTest.php

```diff
@@ -273,7 +273,10 @@ public function testDeferredCond() {
     $this->mockResolver('Query', 'tree', $this->builder->fromValue(['name' => 'some tree', 'id' => 5]));
     $this->mockResolver('Tree', 'name', $this->builder->cond([
       [$this->builder->fromValue(FALSE), $this->builder->fromValue('This should not be in the result.')],
-      [function () { return new Deferred(function () { return TRUE; }); }, $this->builder->fromValue('But this should.')],
+      [function () {return new Deferred(function () {return TRUE;
```

Comment:
let's format these return statements on the next lines with 2 space additional indentation.